src/sql/util.py

```diff
@@ -494,3 +499,83 @@ def get_default_configs(sql):
     del default_configs["parent"]
     del default_configs["config"]
     return default_configs
+
+
+def parse_connect_arg(connect_arg):
```

Comment:
I'm unsure what this code is for. we just need to show the widget when running `%sqlcmd connect`, what are these functions for?

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/783/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R13)

I forgot to delete this. It was used for adding connection details to connections.ini when a user runs `%sql duckdb://`